.github/workflows/ghidra-buildbot.yml

```
@@ -0,0 +1,111 @@
+name: Ghidra
+on: push
```

Maybe this is a good idea, but that's something for you guys to decide. Maybe in future commits.
```suggestion
on: [push, pull_request]
```